ragtime.jdbc/src/ragtime/jdbc.clj

```diff
@@ -18,10 +18,13 @@
                          [:created_at "varchar(32)"]]))
 
 (defn- get-table-names [db-spec]
-  (-> (sql/get-connection db-spec)
-      (.getMetaData)
-      (.getTables nil nil "%" nil)
-      (sql/metadata-result {:row-fn :table_name})))
+  (sql/with-db-connection
```

Comment:
This could be replaced with:

```clojure
(with-open [conn (sql/get-conneciton db-spec)]
   ...)
```

As there's little point in putting the connection inside a db-spec map if we're just going to take it out again.